src/libcollections/btree/map.rs

```diff
@@ -1809,58 +1778,39 @@ fn handle_underfull_node<'a, K, V>(node: NodeRef<marker::Mut<'a>,
         Merged(handle.merge().into_node())
     } else {
         if is_left {
-            Stole(handle.steal_left().into_node())
+            handle.steal_left();
+            Stole(handle.into_node())
```

Comment:
This return is the same, regardless of whether `is_left` is true or not, so it can be moved out of the `if` statement.